### PR TITLE
Update deps and changelog

### PR DESCRIPTION
**Pull request by plorenz**

### CODE REVIEW

File: etc/ctrl.with.edge.yml [hunk 1]
```diff
@@ -55,6 +55,7 @@ metrics:
 #events:
 #  jsonLogger:
 #    subscriptions:
+#       - type: fabric.routers
```
Comment (dovholuknf) [inline etc/ctrl.with.edge.yml:58]: nit extra space

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,39 @@
...
+
+## Router events
+
+To enable:
+
+```
+events:
+  jsonLogger:
+    subscriptions:
+       - type: fabric.routers
```
Comment (dovholuknf) [inline CHANGELOG.md:18]: the extra space is here too - yaml gets cranky with that right?

File: etc/ctrl.with.edge.yml (hunk 1 above)
Comment (plorenz) [reply etc/ctrl.with.edge.yml]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (plorenz) [reply CHANGELOG.md]: fixed. When I tested it, it had the extra space and it worked, but doesn't hurt to make it consistent

**Review by scareything — APPROVED** [review]

**Pull request merged**